client/matching/client.go

```diff
@@ -250,10 +274,26 @@ func (c *clientImpl) createLongPollContext(
 	return context.WithTimeout(parent, c.longPollTimeout)
 }
 
-func (c *clientImpl) getClientForTasklist(key string) (Client, error) {
+func (c *clientImpl) getClientForTaskList(key string) (Client, error) {
 	client, err := c.clients.GetClientForKey(key)
 	if err != nil {
 		return nil, err
 	}
 	return client.(Client), nil
 }
+
+func (c *clientImpl) getTaskListsByDomain(
+	cl Client,
+	ctx context.Context,
+	request *types.MatchingGetTaskListsByDomainRequest,
+	opts ...yarpc.CallOption,
+) (*types.GetTaskListsByDomainResponse, error) {
+	ctx, cancel := c.createContext(ctx)
+	defer cancel()
+
+	resp, err := cl.GetTaskListsByDomain(ctx, request, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return resp, nil
```

Comment:
nit: can be simplified to `return cl.GetTaskListsByDomain(ctx, request, opts...)` ?

Reply:
updated